packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/webview_credential.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:meta/meta.dart';
```

Comment:
It looks like we missed this in the other PR. This was missing a license.

```suggestion
// Copyright 2013 The Flutter Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

import 'package:meta/meta.dart';
```